src/customer-service/Factories/EmployeeFactory.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Data.Sqlite;
+using customer_service.Models;
+using customer_service.Data;
+
+namespace customer_service.Factories
```

Comment:
Do we want to segment the namespace? I guess it doesn't hurt, I just want everyone to be consistent.